.mergify.yml

```diff
@@ -1,7 +1,3 @@
-# Validate your changes with:
-#
-#   $ curl -F 'data=@.mergify.yml' https://gh.mergify.io/validate/
```

Comment:
This seems to be gone now, I couldn't find a replacement in the docs after a bit of searching unfortunately so I just removed it.